src/explore.c

```diff
@@ -186,11 +276,12 @@ bool explore_debug(sqlite3 *db) {
     return true;
 }
 
-bool explore_cmd(sqlite3 *db, FILE *output) {
+bool explore_cmd(sqlite3 *db, FILE *output, uint8_t mode) {
```

Comment:
This change allows the explorer to work in either `EXPLORE` mode or `PRUNE` mode